sky/global_user_state.py

```diff
@@ -264,6 +264,22 @@ class ClusterEventType(enum.Enum):
     sqlalchemy.Column('updated_at', sqlalchemy.Integer),
 )
 
+cluster_failures_table = sqlalchemy.Table(
+    'cluster_failures',
+    Base.metadata,
+    sqlalchemy.Column('id',
+                      sqlalchemy.Integer,
+                      primary_key=True,
+                      autoincrement=True),
+    sqlalchemy.Column('cluster_hash', sqlalchemy.Text, index=True),
+    sqlalchemy.Column('failure_mode', sqlalchemy.Text, index=True),
+    sqlalchemy.Column('failure_reason', sqlalchemy.Text),
+    sqlalchemy.Column('cleared_at',
+                      sqlalchemy.Integer,
+                      server_default=None,
+                      index=True),
+)
```

Comment:
we can add a brief comment for failure_mode, failure_reason, and cleared_at explaining what they are